helm_deploy/values-prod.yaml

```diff
@@ -2,10 +2,22 @@
 # Per environment values which override defaults in hmpps-notifications-alerts-vsip/values.yaml
 
 generic-service:
+  replicaCount: 2
```

Comment:
4 on prod?

Reply:
do you think we need 4 on prod,... this is not a api or web service,... should not have too much of a load because it uses Queues...